app/styles/_log.less

```diff
@@ -203,18 +203,25 @@
   bottom: 5px;
   left: 10px;
 }
+
 .chromeless {
   background-color:  @log-bg-color;
   color: #d1d1d1;
-  .log-pending-ellipsis {
-    background-color: @log-bg-color;
-    padding: @grid-gutter-width 0 0 (@grid-gutter-width - 10px);
-  }
-  .log-scroll-top.affix {
-    right: 0px; // override
+  .log-view {
+    .log-pending-ellipsis {
+      background-color: @log-bg-color;
+      padding: @grid-gutter-width 0 0 (@grid-gutter-width - 10px);
+    }
+    .log-scroll-top.affix {
+      right: 0px; // align to edge of log-view when showScrollLinks is false
+      &.log-has-scroll {
+        right: @scrollbar-width; // align to edge of scrollbar when showScrollLinks is true
+      }
+    }
   }
-  .log-scroll-top.affix.target-logger-node {
-    right: @scrollbar-width;  // override
+  ::-webkit-scrollbar-track {
```

Comment:
Would love to see this moved to _scrollbars.less since it's specific to scrollbar styling.